android/player/src/main/java/com/intuit/playerui/android/asset/DecodableAsset.kt

```diff
@@ -114,9 +114,13 @@ public abstract class DecodableAsset<Data>(assetContext: AssetContext, private v
         }
 
         override fun apply(androidPlayer: AndroidPlayer) {
-            androidPlayer.onUpdate { _, _ ->
-                synchronized(trackedHydrations) {
-                    trackedHydrations.clear()
+            androidPlayer.hooks.viewController.tap { viewController ->
+                viewController?.hooks?.view?.tap { view ->
+                    view?.hooks?.onUpdate?.tap { _ ->
+                        synchronized(trackedHydrations) {
+                            trackedHydrations.clear()
+                        }
+                    }
```

Comment:
Unfortunately, `androidPlayer.onUpdate` triggers asset decoding logic, which isn't needed for this plugin and causes some redundant logs. This is more verbose, but more targeted and trims redundant operations. Really, it would be great to encapsulate the asset decoding lifecycle as another hook to tap.